pkg/state/state.go

```diff
@@ -1054,7 +1054,7 @@ func (st *HelmState) PrepareCharts(helm helmexec.Interface, dir string, concurre
 			return nil, []error{err}
 		}
 	} else {
-		selected = st.Releases
+		selected = st.GetReleasesWithOverrides()
```

Comment:
just curious -- did you find this using a go debugger or just by reading the code?

Reply:
Both.